rpcs3/Emu/RSX/VK/VKFragmentProgram.cpp

```diff
@@ -228,43 +228,17 @@ void VKFragmentDecompilerThread::insertMainStart(std::stringstream & OS)
 		{
 			if (m_prog.front_back_color_enabled)
 			{
-				if (PI.name == "spec_color")
+				if (PI.name == "spec_color" && m_prog.back_color_specular_output && m_prog.front_color_specular_output)
```

Comment:
I'm not sure at all, I think that if a color is not exported by vertex shader then the value should be vec4(0., 0., 0., 1.).